packages/gitlab/manifest.yml

```
@@ -0,0 +1,37 @@
+format_version: 3.1.3
+name: gitlab
+title: Gitlab
```

The screen shot made me see some more things.
```suggestion
title: GitLab
```